### PR TITLE
Feature/game ui

### PR DESCRIPTION
**Pull request by dotconde**
# Content

* Statement component
* Image Container for game stage
* Titles for gameUI
* TaskRow List
* Submit Button

### CODE REVIEW

File: simcase-game/src/components/Statement.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  width: "90%",
+  borderRadius: "4px",
+  padding: "1px 1em",
+  boxSizing: "border-box",
+  marginBottom: "16px",
+  fontSize: "1.2em"
+};
+const textStyles = {
+  color: "green",
+  fontFamily: "Arial"
```
Comment (CesarCachay) [inline simcase-game/src/components/Statement.js:19]: What about changing this line to the const called Container?
I think that would help you adding that font family of the button you are working

File: simcase-game/src/components/Statement.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+
+const Container = {
+  textAlign: "center",
+  backgroundColor: "rgb(213,245,245)",
+  marginLeft: "auto",
```
Comment (test0n3) [inline simcase-game/src/components/Statement.js:8]: Th margin lines can be reduced to: ```margin: "0 auto 16px"```

File: simcase-game/src/components/TaskRow.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  fontSize: "0.9em"
+};
+
+function TitlesForm({ task }) {
+  return (
+    <div css={container}>
+      {/* Task name */}
+      <div css={item}>{task}</div>
+
+      {/* Radio buttons 🍎*/}
```
Comment (CesarCachay) [inline simcase-game/src/components/TaskRow.js:34]: What do you thing about deleting this commented line?

File: simcase-game/src/components/Statement.js (hunk 1 above)
Comment (test0n3) [inline simcase-game/src/components/Statement.js:19]: Not all systems include this font. Also a fallback font is required, likes `sans-serif`

File: simcase-game/src/components/TitlesForm.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  background: "rgb(213,245,245)",
+  borderRadius: "4px",
+  display: "table",
+  tableLayout: "fixed",
+  width: "90%",
+  height: "4vh",
+  boxSizing: "border-box",
+  marginLeft: "auto",
+  marginRight: "auto",
+  marginTop: "16px", //Needs a little fix
```
Comment (CesarCachay) [inline simcase-game/src/components/TitlesForm.js:15]: This line and the one bellow has comments. Are they necessary?

File: simcase-game/src/components/TaskRow.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  textAlign: "center",
+  fontFamily: "Arial",
+  color: "rgb(61,125,255)",
+  fontSize: "0.9em"
+};
+
+function TitlesForm({ task }) {
+  return (
+    <div css={container}>
+      {/* Task name */}
```
Comment (CesarCachay) [inline simcase-game/src/components/TaskRow.js:31]: This line is commented. Is this one necessary?

**Review by CesarCachay — APPROVED** [review]

File: simcase-game/src/components/TaskRow.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+const container = {
+  background: "rgb(213,245,245)",
+  borderRadius: "4px",
+  display: "table",
+  tableLayout: "fixed",
+  width: "90%",
+  height: "4vh",
+  boxSizing: "border-box",
+  marginLeft: "auto",
```
Comment (test0n3) [inline simcase-game/src/components/TaskRow.js:13]:
```
margin: "16px auto"
```

**Review by test0n3 — APPROVED** [review]

**Pull request merged**